packages/vm/src/evm/opcodes/codes.ts

```diff
@@ -249,6 +249,16 @@ const hardforkOpcodes: { hardforkName: string; opcodes: OpcodeEntry }[] = [
 ]
 
 const eipOpcodes: { eip: number; opcodes: OpcodeEntry }[] = [
+  {
+    // These numbers differ than what is currently implemented
+    // in geth due to other eips being included here that are
+    // not in geth
```

Comment:
Right, the EIP also does not yet state whatever opcodes numbers we should use.

Reply:
We now have some numbers that we can use: https://github.com/ethereum/EIPs/commit/b6b3857fbcec36048e527ebb5e0eb8b582f6c96c